### ISSUE TITLE
adapt sea-map run-scripts for npm v7+

### ISSUE THREAD

**Issue by wu-lee** [DigitalCommons/mykomap#143]
The NPM maintainers, in their wisdom, have decided to remove a feature we use in our run-scripts for building and deploying.  As such, we can't use versions of NPM newer than 6 (7 or higher) until we find a way around this missing feature.

See:
https://github.com/npm/cli/issues/2452#issuecomment-1028142496
https://github.com/npm/rfcs/issues/305

**Comment by ColmMassey** [comment]
When might we need to address this? @wu-lee

**Comment by wu-lee** [comment]
I've needed to bump the version of node in use to a recent one (16.*, i.e. `lts/gallium`) in order to get an implementation of `Promise.all`, used in the sparql-bypass feature to await several asynchronous download operations in parallel.

Therefore, I've run into this problem again.  But this time I've found a work-around: use `npm config get [..]` to obtain the config value from `.npmrc` and then set the appropriate environment variable, in the `bin/sea-site-build` script.

Fix applied in 4d9020074ea7
